lib/dockerize_rails/constants.rb

```diff
@@ -31,20 +31,28 @@ module Constants
           :'--test-env' => 'generates configurations to run tests.'.freeze
         }
       },
-      docker_info: {
-        aliases: %I[docker_info di],
-        help: 'Shows Docker information'.freeze
-      },
       undockerize: {
         aliases: %I[undockerize ud du u dd],
         help: 'Removes docker configurations'.freeze,
         params: {
           :'--purge' => "also removes #{Constants::DOCKERIZE_RAILS_CONFIG_FILE_NAME}".freeze
         }
       },
+      docker_info: {
+          aliases: %I[docker_info di],
+          help: 'Shows Docker information'.freeze
+      },
+      docker_pull: {
+          aliases: %I[docker_pull pull dp],
+          help: 'Pulls Docker images'.freeze
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.

<!-- Comment by [SideCI](https://sideci.com) -->